jekyll/_cci2/insights-glossary.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: classic-docs
+title: "Insights metrics glossary"
+short-title: "Insights metrics glossary"
+short-title: "Metric definitions for CircleCI insights"
+version:
+- Cloud
+---
+
+## Overview
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 2. You can fix by inserting `{: #overview }` to the next line